tests/dummy/config/environment.js

```diff
@@ -18,6 +18,10 @@ module.exports = function(environment) {
       // when it is created
     },
 
+    'ember-metrics': {
+      includeAdapters: ['segment', 'google-tag-manager']
+    },
+
     metricsAdapters: [
```

Comment:
Yeah, we can deprecate the use of `metricsAdapters` and issue a deprecation warning in the metrics initializer. I much prefer using `ember-metrics` as a config key, since it's more explicit about which addon the config comes from. I'm also planning to update the config API so it's less repetitive (in a future PR), to something like:

```js
'ember-metrics': {
  includeAdapters: // ...,
  adapterOptions: [
    {
      name: 'GoogleAnalytics',
      development: {
        id: 'UA-XXXX-Y'
      },
      test: false,
      production: {
        id: 'UA-YYYY-X'
      }
    }
  ]
}
```